### ISSUE TITLE
Feasibility: Assess viability of automatic updates to documentation

### ISSUE THREAD

**Issue by Aveline-art** [hackforla/website#2228]
### Dependency
- #3636
- #2676

### Overview
As a developer, we need documentation that is up-to-date and easy to maintain that way. In this issue, we will be performing a feature assessment on having automatic updates on the wiki and our contributing documentation.

### Action Items

- [ ] Assess whether a program can:
  - [x] Perform edits to files within a repository (please test this on your own repo)
  - [x] Perform edits to a wiki (please test this in your own wiki)
  - [ ] If either is possible, list out the permissions that would be required by any API tokens, and any technology that we need
- [ ] Create a demo that can parse and read meetings times from our [vrms api](https://www.vrms.io/api/recurringevents) and display it in a table on the wiki.

### Resources/Instructions
Currently, most of our documentation is outdated, or defunct, and this presents a really big problem for new members of the team. While most documentation must be manually reviewed, if it can be automatically updated, even if partially, it would be a big help to the team. For example, we would love to have a program update a wiki about labels or milestones as we add, remove, or edit them!

The mechanism through which automatic updates can be done probably involves these steps:

1. download the wiki (the wiki is [a repo](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis))!
2. read a file in the wiki
3. make changes!
4. push the repo back

When making a demo, you can also look at the [gollum trigger for GHAs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#gollum) and attempt to ~~steal the one ring that rules them all~~ create a demo using a GHA.

[About wikis](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis)
[Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows)
[Wiki home](https://github.com/hackforla/website/wiki)
[website/CONTRIBUTING.md ](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)

**Comment by JessicaLucindaCheng** [comment]
This issue needs to be edited by a lead to reflect our migration to the [website-wiki repo](https://github.com/hackforla/website-wiki). For more info, read [How to Contribute to the Wiki](https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki).

**Comment by vincentdang** [comment]
@JessicaLucindaCheng I'm curious to know more about this issue.

**Comment by JessicaLucindaCheng** [comment]
@vincentdang I don't actually know much about it. I left a comment because of the issue in the Dependency section. Sorry.

**Comment by vincentdang** [comment]
@JessicaLucindaCheng oh okay. I was debating on whether I should take it on.

**Comment by JessicaLucindaCheng** [comment]
@roslynwythe

- I saw this issue was moved directly from the Ice Box to the Prioritized Backlog and I'm not sure if it was determined this issue didn't need edits. I left the following comment when I moved this issue to the Ice Box https://github.com/hackforla/website/issues/2228#issuecomment-1484398262:
> This issue needs to be edited by a lead to reflect our migration to the [website-wiki repo](https://github.com/hackforla/website-wiki). For more info, read [How to Contribute to the Wiki](https://github.com/hackforla/website/wiki/How-to-Contribute-to-the-Wiki).

  - Since this issue was written before the wiki migration to MkDocs, does this issue need to be edited to reflect the migration?
  - Also, this issue may need clarification because a developer, Vincent, reached out to me asking me for more information about this issue (see comments in this issue) and I didn't have more info than what is written in the issue itself.
  - In addition, is the website-wiki repo ready for developers to start contributing to it? If not, this may need to be put back in the "Ice Box" and the "Dependency" section needs to be edited for what needs to be done in order for this issue to come out of the "Ice Box".

**Comment by ramitaarora** [comment]
i. Availability: 9am-12pm Weekdays
ii. ETA: April 30

**Comment by ramitaarora** [comment]
Currently blocked: I have forked the repo and made a new branch to make edits. I see the reflected changes in the branch when using docker and locally hosting it - am I supposed to see it change on the Wiki tab on the GitHub fork? Currently, the Wiki is empty.

https://github.com/ramitaarora/hfla-website-wiki/tree/feasibility-2228

Thanks so much!

@t-will-gillis I was suggested to ask you for help!

**Comment by t-will-gillis** [comment]
Hey @ramitaarora I am going to ask @JessicaLucindaCheng and @roslynwythe to weigh in on this. I am not sure how moving the Wiki to MkDocs affects this issue.

**Comment by roslynwythe** [comment]
@ramitaarora I don't believe it will be possible to work on this issue right now because the process and documentation for updating the new wiki are not in place, and I am not certain how long that will take, so I need to unassign you and apply a dependency to the issue.  I apologize for not catching the error sooner.

**Comment by ramitaarora** [comment]
@roslynwythe Thanks Roslyn! No problem at all.
